spokestack/tts/tts_manager.py

```diff
@@ -0,0 +1,35 @@
+"""
+This module contains the Spokestack text to speech manager which handles a
+text to speech client, decodes the returned audio, and writes the audio to
+the specified output
+"""
+from streamp3 import MP3Decoder  # type: ignore
+
+
+class TextToSpeechManager:
+    """ Manages tts client and outputs target
+
+        Args:
+            client: Text to speech client that returns encoded mp3 audio
+            output: Audio outputs target
+    """
+
+    def __init__(self, client, output) -> None:
+        self._client = client
+        self._output = output
+
+    def synthesize(self, **kwargs) -> None:
```

Comment:
tts parameters should be explicit here - they can't vary across tts clients, and you want the interface to be discoverable to applications